beacon-chain/state/interfaces.go

```diff
@@ -65,6 +65,7 @@ type ReadOnlyBeaconState interface {
 	LatestExecutionPayloadHeader() (interfaces.ExecutionData, error)
 	LastWithdrawalValidatorIndex() (types.ValidatorIndex, error)
 	ExpectedWithdrawals() ([]*enginev1.Withdrawal, error)
+	NextWithdrawalIndex() (uint64, error)
```

Comment:
That's the third getter related to withdrawals. How about we extract them to a `type ReadOnlyWithdrawals interface`?